### PR TITLE
Change committ flow

### PR DESCRIPTION
**Pull request by danielochuba**
I am making a request to merge this code as changes to the existing code base if it passes the core requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,5 @@
 <a name="readme-top"></a>
```
Comment (1GORDON) [inline README.md:2]: Great job @danielochuba 🤝

- [ ] As stated in the project requirements, kindly deploy your project using GitHub Pages, make sure all pages, links, and images work as expected, and add the link of the deployed project to your README.

Comment (1GORDON) [inline README.md:2]: Great job @danielochuba 🤝

- [ ] As stated in the project requirements, make sure your README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: css/mobileStyle.css [hunk 2]
```diff
@@ -229,7 +229,6 @@ body {
   margin: auto;
```
Comment (1GORDON) [inline css/mobileStyle.css]: Great design @danielochuba 🤝

- [ ] As stated in the project requirements. kindly stick to the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as much as possible. Kindly implement these changes across desktop and mobile screens.
- [ ] Kindly make sure your design contains two navbars as seen in the given design.
- [ ] Kindly make sure the general background colors, fonts, and font sizes are replicas of the given design.
- [ ] Kindly make sure your design is responsive on both the desktop and the mobile screens. Some parts of the design shrink and disappear on the mobile view.
- Your Main Design
desktop-view
- Given Design
Given-design
- Given program
given-program
- Your Program
your-program

File: css/mobileStyle.css [hunk 3]
```diff
@@ -229,7 +229,6 @@ body {
...
 }
 
-
 /* ABOUT PAGE STYLES */
```
Comment (1GORDON) [inline css/mobileStyle.css]: Great design @danielochuba 🤝

- [ ] As stated in the project requirements, kindly stick to the design guidelines as much as possible. You may make sure your about footer has a dark background and kindly reduce its size.
- Your About Footer
about-footer
- Given About Footer
given-about-footer

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @danielochuba ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] As stated in the project requirements, kindly record a video presenting the main features of the project and remember to highlight a piece of the code you find interesting or proud of making.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/mobileStyle.css (hunk 3 above)
Comment (danielochuba) [reply css/mobileStyle.css]: @1GORDON thanks, but i need to know if the project is okay before making a video

**Comment by danielochuba** [comment]
# Please Am so Sorry, Kindly help me through #

## After the first review, I thought it would be better to build a new project that strictly follows the template, but then i mistakenly merged the branch, don't  be offended please help me through##

Please I had to rebuild the project entirely
[https://github.com/danielochuba/HTML-basic-JavaScript-Capstone/pull/6](https://github.com/danielochuba/HTML-basic-JavaScript-Capstone/pull/6) is now the actual pull request.

**Comment by uwadonat** [comment]
Review was performed on this PR:
https://github.com/danielochuba/HTML-basic-JavaScript-Capstone/pull/6

**Pull request merged**